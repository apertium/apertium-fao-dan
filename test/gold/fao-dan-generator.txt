[1ovW7Sfu3fa5] hans pige
[/1ovW7Sfu3fa5]
