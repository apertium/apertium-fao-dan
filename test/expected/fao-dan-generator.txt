[1ovW7Sfu3fa5#0] hans pige
[/1ovW7Sfu3fa5]
